packages/ember-htmlbars/tests/helpers/if_unless_test.js

```diff
@@ -205,67 +205,6 @@ QUnit.test('edge case: rerender appearance of inner virtual view', function() {
   equal(jQuery('#qunit-fixture').text(), 'test');
 });
 
-QUnit.test('`if` helper with inline form: renders the second argument when conditional is truthy', function() {
```

Comment:
:metal: (this should be covered by the generic tests once we have resolved https://github.com/emberjs/ember.js/pull/12920/files#r53242487)